docs/packages/pkg/libuv.rst

```diff
@@ -22,7 +22,7 @@ libuv
 
     hunter_add_package(libuv)
     find_package(libuv CONFIG REQUIRED)
-    target_link_libraries(... libuv::uv)
+    target_link_libraries(... uv)
```

Comment:
where did the imported namespace go?

Reply:
yea offical libuv repro is not defining namespace libuv:: but the hunter repro was doing it.